client/app/templates/account.jade

```diff
@@ -102,6 +102,13 @@
                         .account-form-error-text= t('account 2fa error')
 
             #2fa-infos.account-field
+                p#2fa-strategy
+                
+                p.bold= t('account 2fa rectokens')
```

Comment:
I know them. But these classes already exists, and I find it cleaner to use them instead of the said tags :smile: